onnxruntime/core/graph/function.cc

```diff
@@ -268,12 +268,12 @@ FunctionImpl::FunctionImpl(const onnxruntime::Graph& graph,
     }
     int i = 0;
     for (auto& input : cached_op_schema->inputs()) {
-      op_schema_->Input(i, input.GetName(), input.GetDescription(), input.GetTypeStr());
+      op_schema_->Input(i, input.GetName(), input.GetDescription(), input.GetTypeStr(), input.GetOption());
```

Comment:
How about is_homogeneous and min_arity? Any harm to add them ?

Reply:
sure, I add options because it was actually a bug and causing segmentation fault. 

---
In reply to: [411612229](https://github.com/microsoft/onnxruntime/pull/3567#discussion_r411612229) [](ancestors = 411612229)